scss/components/_legal-search.scss

```diff
@@ -45,14 +45,30 @@
   }
 }
 
-.legal-mur__archive {
-  font-size: u(1.3rem);
-  margin-top: u(0.8rem);
-
-  .legal-mur__archive-icon {
-    @include u-icon($file-box, $base, 1.5rem, 1.5rem, 100%);
-    display: inline-block;
-    margin-right: u(0.7rem);
-    vertical-align: top;
+.legal-mur {
+  .legal-mur__archive {
+    font-family: $sans-serif;
+    font-size: u(1.3rem);
+    margin-top: u(0.8rem);
+
+    .legal-mur__archive-icon {
+      @include u-icon($file-box, $base, 1.5rem, 1.5rem, 100%);
+      display: inline-block;
+      margin-right: u(0.7rem);
+      vertical-align: top;
+    }
+  }
+
+  .legal-mur__property {
+    margin-bottom: u(1rem);
+  }
+
+  .legal-mur__subject-list {
```

Comment:
[.list--bulleted](https://github.com/18F/fec-style/blob/master/scss/components/_list-styles.scss#L23) actually does more than just the margins, it also adds padding and font-size. I can clean that up though, maybe move that into a `.list--large` or `.list--spaced` ?